### PR TITLE
Report number of mutations done by dataflow.

### PR DESCRIPTION
**Pull request by mgarolera**

### CODE REVIEW

File: bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableIO.java [hunk 1]
```diff
@@ -736,9 +738,14 @@ public static Pipeline initializeForWrite(Pipeline p) {
...
     private final String tableName;
 
+    // Stats
+    private final Aggregator<Long, Long> mutations;
+
     public CloudBigtableSingleTableBufferedWriteFn(CloudBigtableTableConfiguration config) {
       super(config);
       this.tableName = config.getTableId();
+
+      mutations = createAggregator("mutations", new Sum.SumLongFn());
```
Comment (garye) [inline bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableIO.java]: this.mutations for consistency (same comment for each Fn)

Comment (mgarolera) [inline bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableIO.java]: Done. PTAL

-- 

Marçal

**Comment by sduskis** [comment]
+1 on aggregators.  We should do this for the other Writer DoFns, the Reader and the Source splitting.  The performance logging in AbstractCloudBigtableTableDoFn should be removed.

File: bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableIO.java (hunk 1 above)
Comment (garye) [inline bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableIO.java]: Cool, thanks

**Pull request merged**